.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules
 .settings
 *.log
 dist/
+.idea
```

Comment:
Things like this should be handling via [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).